palette/src/alpha/alpha.rs

```diff
@@ -44,6 +44,23 @@ impl<C, T: Component> Alpha<C, T> {
     }
 }
 
+impl<C, T> PartialEq for Alpha<C, T>
+where
+    T: Component + PartialEq,
```

Comment:
I bet the `Component` here is what makes it require them for the other comparison traits. It shouldn't be necessary here and for `Eq`. :slightly_smiling_face: